scene/3d/vehicle_body.cpp

```diff
@@ -559,13 +559,13 @@ void VehicleBody::_resolve_single_bilateral(PhysicsDirectBodyState *s, const Vec
 			b2invinertia,
 			b2invmass);
 
-	real_t rel_vel = jac.getRelativeVelocity(
+	jac.getRelativeVelocity(
 			s->get_linear_velocity(),
 			s->get_transform().basis.transposed().xform(s->get_angular_velocity()),
 			b2lv,
 			b2trans.xform(b2av));
 
-	rel_vel = normal.dot(vel);
+	real_t rel_vel = normal.dot(vel);
```

Comment:
I think for now it's best if you leave the code as it was, and add a FIXME comment. My guess *might* be correct, but it's best fixed in its own commit for clarity anyway, after checking that it doesn't not break VehicleBody.